frontend/tailwind.config.js

```
@@ -8,9 +8,16 @@ module.exports = {
                 fontFamily: {
-                        sans: ['Space Grotesk', 'ui-sans-serif', 'system-ui', 'sans-serif'],
+                        sans: ['Outfit', 'ui-sans-serif', 'system-ui', 'sans-serif'],
+                        body: ['IBM Plex Sans', 'ui-sans-serif', 'system-ui', 'sans-serif'],
                         mono: ['JetBrains Mono', 'ui-monospace', 'monospace'],
                 },
+                colors: {
+                        ink: '#050505',
+                        midnight: '#0A1128',
+                        'matte-grey': '#8C92AC',
+                        silver: '#E2E8F0',
+                },
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)